Doc/c-api/init_config.rst

```diff
@@ -1602,6 +1602,53 @@ customized Python always running in isolated mode using
 :c:func:`Py_RunMain`.
 
 
+Get the current Python configuration
+====================================
+
+Get a configuration option where *name* is the name of a :c:type:`PyConfig`
+member.
+
+Some options are read from the :mod:`sys` attributes. For example, the option
+``"argv"`` is read from :data:`sys.argv`.
+
+
+.. c:function:: int PyConfig_Get(const char *name, PyObject **value)
+
+   Get a configuration option as a Python object.
+
+   The object type depends on the configuration option. It can be:
+
+   * ``int``
+   * ``str``
+   * ``list[str]``
+   * ``dict[str, str]``
+
+   * Return ``0`` and set *\*value* on success.
+   * Raise an exception and return ``-1`` on error.
```

Comment:
If the result is an error or a `PyObject*`, then the return type can just be `PyObject *` and NULL on error. (And I guess worth mentioning that https://github.com/capi-workgroup/api-evolution/issues/13 is still a discussion and not a guideline yet.)

Reply:
Oops sorry, I changed the API multiple times, there were many back and forth. At the end, I forgot to update the doc! Fixed. The function does return a Python object (`PyObject*`), or NULL (and set an exception) on error.